src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,42 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector =
+            Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+
+        MovieService movieService = (MovieService) injector
+                .getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService = (CinemaHallService)
+                injector.getInstance(CinemaHallService.class);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(500);
+        cinemaHall.setDescription("Big hall");
+        cinemaHallService.add(cinemaHall);
+        System.out.println(cinemaHallService.getAll());
+
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setId(movieSession.getId());
+        movieSessionService.add(movieSession);
```

Comment:
The `MovieSession` object is being created without setting essential properties such as the movie, cinema hall, and show time. Ensure that these properties are set before adding the session to the service to avoid incomplete data entries.